addons/web/static/src/views/fields/statusbar_time/statusbar_time_field.scss

```diff
@@ -0,0 +1,28 @@
+.o_field_statusbar_time {
+    @extend .o_field_statusbar;
+    .time-btn {
+        color: $primary;
+        cursor: pointer;
+        font-size: 0.85em;
+    }
+    .time-btn-hover {
+        opacity: 0.2;
+    }
+    .time-btn-hover:hover {
+        opacity: 1;
+    }
+    .time-tracking {
+        color: #898282c2;
+        font-size: .85em;
+    }
+    .time-btn-container {
+        display: flex;
```

Comment:
we now use bootstrap classes intensively, to reduce the css size. You can do almost everything with bootstrap context classes.